src/platform/qt/ShaderSelector.cpp

```diff
@@ -21,7 +21,7 @@
 #include <mgba-util/vfs.h>
 #include "platform/video-backend.h"
 
-#if !defined(_WIN32) || defined(USE_EPOXY)
+#if defined(BUILD_GL) && (!defined(_WIN32) || defined(USE_EPOXY))
```

Comment:
Pretty sure this check is wrong. Needs to be BUILD_GLES2 || (WIN32 && USE_EPOXY) IIRC. Would need to test. Check out DisplayGL.cpp for a related example.